assets/js/socketSimulator.js

```diff
@@ -0,0 +1,68 @@
+var _ = require('underscore');
+
+var socket = require('socket-io-mock');
+var server = new socket();
+
+server.socketClient.connected = true;
+
+server.on('get', function(payload) {
+  //console.log(payload);
```

Comment:
I've removed this line, and performed a direct commit via Github into the same PR.